src/repo/__tests__/github.js

```diff
@@ -65,6 +65,10 @@ test('handle errors', async () => {
   await rejects(getUserInfo('nodisplayname'))
 })
 
+test('Throw error when no username is provided', async () => {
+    expect(await getUserInfo()).toThrow()
```

Comment:
Can anyone please help me with:

```suggestion
    expect(await getUserInfo()).toThrow()
```

Even though the function throws error, the test still fails as if throwing error shouldn't pass the test. @Berkmann18 @kentcdodds

Reply:
you can try about that.
```
test('Throw error when no username is provided', async () => {
    try {
      await getUserInfo();
    } catch (e) {
      expect(e.message).toEqual('No login when adding a contributor. Please specify a username.');
    }
})
```